### ISSUE TITLE
Support multiple values per field

### ISSUE THREAD

**Issue by nightexcessive** [potatolondon/search#48]
The search API allows multiple fields with the same name, effectively meaning that the field has multiple values<sup>[1][ref1]</sup>. In Django nomenclature, this would be functionally equivalent to a ListField. search should support this.

In my case, I have a Djangae app where I'd like to allow a TextField to have multiple values.

[ref1]: https://cloud.google.com/appengine/docs/python/search/#document_fields

**Comment by davidwtbuxton** [comment]
Yes, I want this for a project where we have lots of list fields and related set fields where I want to search for records that have 1 or more of a set of values.

I haven't looked into how difficult it would be to add support for this, was inclined to move away from search and just use the Google API directly (but then I'm more familiar with that API).

**Comment by davidwtbuxton** [comment]
There was previous work related to this feature in #14 and #15.
